.gitmodules

```diff
@@ -15,28 +15,28 @@
 [submodule "kubelet"]
 	path = kubelet
 	url = https://github.com/openshift/kubernetes
-	branch = release-4.17
+	branch = release-4.18
 [submodule "kube-proxy"]
 	path = kube-proxy
 	url = https://github.com/openshift/kubernetes
 	branch = master
```

Comment:
not asking to change this, but is there a reason kube-proxy points to master and kubelet to 4.18?

Reply:
this was done on recommendation of SDN team. Since we tie kube-proxy to upstream tags, not sure there is a difference